src/Debug/Stopwatch/Connection.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace atk4\dsql\Debug\Stopwatch;
+
+use atk4\dsql\Connection_Proxy;
+use atk4\dsql\Expression;
+
+class Connection extends Connection_Proxy
+{
+    public $driverType = 'stopwatch';
+
+    /**
+     * Callable to call for outputting.
+     *
+     * Will receive parameters:
+     *  - Expression Expression object
+     *  - float      How long it took to execute expression
+     *  - boolean    True if we had exception while executing expression
+     *
+     * @var callable
+     */
+    public $callback;
+
+    /**
+     * @var float
+     */
+    protected $startTime;
+
+    public static function createDriver(array $dsn)
```

Comment:
there is 2x `createDriver` - should be probably part of later PR when all other Connection NSes are updated and if we choose to use the name `Driver`.

Reply:
Agree